package.json

```diff
@@ -96,7 +96,6 @@
     "@types/jest": "^29.5.5",
     "@types/jest-axe": "^3.5.6",
     "@types/js-yaml": "^4.0.6",
-    "@types/marked": "^5.0.2",
```

Comment:
Added a commit to remove `@types/marked`

They've been published with the package for a while